features/channels/create_new_channel.feature

```diff
@@ -0,0 +1,17 @@
+@246
+Feature: Create new channels
```

Comment:
If we are going to implement #246 in Sprint 8, replace all `Channel` with `Topic`